main.go

```diff
@@ -307,7 +307,7 @@ func validateParams() providerConfig {
 	var ecrEnabled bool
 
 	awsAccountID = os.Getenv("awsaccount")
-	if len(awsAccountID) == 0 {
+	if len(awsAccountID) == 0 || awsAccountID == "changeme" {
```

Comment:
Do you think we should merge the secrets? Then make it more explicit which cloud is desired? I want to get everyone working, but may refactor to have it try and work with any cloud at the same time (or both).